application-mocca-calendar-ui/src/main/resources/MoccaCalendar/Code/Macro.xml

```diff
@@ -1923,6 +1937,9 @@ Mocca Calendar Macro
     <property>
       <contentType>No content</contentType>
     </property>
+    <property>
+      <defaultCategories/>
+    </property>
     <property>
       <defaultCategory>Content</defaultCategory>
     </property>
```

Comment:
Remove `defaultCategory`. Fill in the `defaultCategories` with the `Content` value.